private/pkg/storage/storagearchive/storagearchive.go

```diff
@@ -219,6 +227,18 @@ func Unzip(
 	return nil
 }
 
+func isAppleExtendedAttributesFile(fileInfo fs.FileInfo) bool {
+	// On macOS, .tar archives created with libarchive will contain additional
+	// files with a prefix of "._" if there are files with extended attributes
+	// and copyfile is enabled.
+	// Archive Utility.app has a similar behavior when creating .zip archives,
+	// except they are placed under a separate MACOSX directory tree.
+	// Here, both are handled by just ignoring all files with a "._" prefix.
+	// This is a reasonable compromise because protobuf IDL files with a "._"
```

Comment:
"because files that live in a Module (`.proto` files, configuration files such as `buf.yaml`, README files) are almost never prefixed with `._`, and fixing this issue in this manner outweighs the slight incorrectness.

Reply:
Done, thanks.